src/main/java/Duke.java

```diff
@@ -1,10 +1,123 @@
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.*;
+import myPackage.*;
```

Comment:
Wildcard imports should be avoided. It might be better to explicitly import the packages you're using.